keyboards/kc60se/config.h

```diff
@@ -55,9 +55,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* number of backlight levels */
 
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
-#define LOCKING_SUPPORT_ENABLE
+// #define LOCKING_SUPPORT_ENABLE
+
 /* Locking resynchronize hack */
-#define LOCKING_RESYNC_ENABLE
+// #define LOCKING_RESYNC_ENABLE
+
+/* debug: emable COMMAND_ENABLE and CONSOLE_ENABLE in rules.mk and run hid_listen */
+#define IS_COMMAND() ( \
+  keyboard_report->mods == (MOD_BIT(KC_LSFT) | MOD_BIT(KC_RSFT)) \
+)
```

Comment:
This can be removed, as it's the default in QMK core.

Reply:
done